templates/bot/csharp/workflow/CardActions/DoStuffActionHandler.cs.tpl

```diff
@@ -0,0 +1,46 @@
+﻿using AdaptiveCards.Templating;
+using Microsoft.Bot.Builder;
+using Microsoft.TeamsFx.Conversation;
+using Newtonsoft.Json;
+using {{SafeProjectName}}.Models;
```

Comment:
Consolidate namespace order in all cs files. (Here project namespace is at bottom but in the handler cs file it's at top...)